applications/ws-affiliation-rnsr/Dockerfile

```
@@ -0,0 +1,9 @@
+FROM inistcnrs/lodex-workers-python:4.0.11
+RUN pip install dvc[all]==2.48 fasttext==0.9.2
```

No need to install *all* remote drivers, as we will only use one.
See [DVC docs](https://dvc.org/doc/install/linux#install-with-pip)

```suggestion
RUN pip install dvc[webdav]==2.48 fasttext==0.9.2
```

---

After using `dvc pull`, don't forget to uninstall `dvc`.